### PR TITLE
Extending the readme

### PR DESCRIPTION
**Pull request by nlindenau**
Added a short description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # nameless-mt-sim
-GIS Development project
```
Comment (nlindenau) [inline README.md:2]: why is it deleted

**Review by lassean — COMMENTED** [review]
looks good

**Review by g-cramberry — COMMENTED** [review]
looks good to me! happy working this way

**Review by g-cramberry — APPROVED** [review]

**Comment by nlindenau** [comment]
thanks

**Comment by kaisasavioja** [comment]
looks nice!

**Pull request merged**